packages/realm/package.json

```diff
@@ -49,14 +49,62 @@
   "scripts": {
     "test": "mocha",
     "test:types": "tsc --project type-tests/tsconfig.json",
-    "build": "rollup -c",
-    "build:watch": "rollup -c --watch",
-    "build:node": "npm run build --prefix ../bindgen",
-    "build:android": "node ../../scripts/build-android.js",
-    "build:ios": "../../scripts/build-ios.sh",
+    "build": "wireit",
+    "bundle": "wireit",
+    "build:node": "wireit",
+    "build:android": "wireit",
+    "build:ios": "wireit",
     "build:node:prebuild": "prebuild --runtime node -- --directory ../bindgen",
     "install": "prebuild-install --runtime node || echo 'Failed to download prebuild for Realm'"
   },
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "bundle",
+        "build:node",
+        "build:android",
+        "build:ios"
+      ]
+    },
+    "bundle": {
+      "command": "rollup --config",
+      "dependencies": [
+        "../realm-network-transport:build",
+        "build:bindgen"
+      ],
+      "files": ["rollup.config.mjs", "src/**/*.ts", "tsconfig.json", "src/**/tsconfig.json"],
+      "output": ["dist/bundle.*.js", "dist/bundle.*.mjs"]
+    },
+    "build:bindgen": {
+      "command": "npm run build --prefix ../bindgen",
+      "files": ["../../src/node/**", "../bindgen/src/**/*.ts", "../bindgen/src/**/*.h"],
```

Comment:
I guess this would be for a later cleanup task, but should we move anything we depend on to build for `node` out of `../../src/node`?